src/components/Explore/TokenDetails/ShareButton.tsx

```diff
@@ -13,6 +13,7 @@ const ShareButtonDisplay = styled.div`
   display: flex;
   cursor: pointer;
   position: relative;
+  z-index: 1;
```

Comment:
could you use a z index value from here? https://github.com/Uniswap/interface/blob/main/src/theme/index.tsx#L29